### PR TITLE
Add UpdateAppList test for cloud apps

### PR DESCRIPTION
**Pull request by jacobkeeler**
ATF Test Scripts to check smartdevicelink/sdl_core#2749

This PR is **ready** for review.

### Summary
Adds UpdateAppList tests for cloud applications

### ATF version
https://github.com/smartdevicelink/sdl_atf/commit/35ca987c2f4e18eb6e2ef09e53cbae4e6a339ce3

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by JackLivio** [comment]
test_scripts/CloudAppRPCs/007_SetCloudAppProperties_InitNewCloudApp.lua

Test fails on this branch.

**Comment by jacobkeeler** [comment]
@JackLivio Fixes

File: test_scripts/CloudAppRPCs/commonCloudAppRPCs.lua [hunk 1]
```diff
@@ -1,6 +1,7 @@
 local actions = require("user_modules/sequences/actions")
 local json = require("modules/json")
 local test = require("user_modules/dummy_connecttest")
+local actions = require("user_modules/sequences/actions")
```
Comment (ShobhitAd) [inline test_scripts/CloudAppRPCs/commonCloudAppRPCs.lua]: Already included above

File: test_scripts/CloudAppRPCs/commonCloudAppRPCs.lua [hunk 2]
```diff
@@ -11,13 +12,17 @@ local function jsonFileToTable(file_name)
   return json.decode(content)
 end
 
-function commonCloudAppRPCs.getCloudAppConfig()
+function commonCloudAppRPCs.getCloudAppConfig(app_id)
   return {
```
Comment (ShobhitAd) [inline test_scripts/CloudAppRPCs/commonCloudAppRPCs.lua]: Add
```
if app_id == nil then app_id=1
```

File: test_scripts/CloudAppRPCs/008_UpdateAppList_NewCloudApplication.lua [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
```
Comment (ShobhitAd) [inline test_scripts/CloudAppRPCs/008_UpdateAppList_NewCloudApplication.lua]: rename the file to `011_UpdateAppList_NewCloudApplication.lua`

File: test_sets/cloudApp_RPCs.txt [hunk 4]
```diff
@@ -4,4 +4,5 @@
 ./test_scripts/CloudAppRPCs/004_CloudAppVehicleId_OnVehicleData.lua
 ./test_scripts/CloudAppRPCs/005_SetCloudAppProperties_Success.lua
 ./test_scripts/CloudAppRPCs/006_SetCloudAppProperties_InvalidParams.lua
-./test_scripts/CloudAppRPCs/007_SetCloudAppProperties_InitNewCloudApp.lua
\ No newline at end of file
+./test_scripts/CloudAppRPCs/007_SetCloudAppProperties_InitNewCloudApp.lua
+./test_scripts/CloudAppRPCs/008_UpdateAppList_NewCloudApplication.lua
```
Comment (ShobhitAd) [inline test_sets/cloudApp_RPCs.txt]: `008` is already in use for GetCloudAppProperties. `009` and `010` are also in use
```suggestion
./test_scripts/CloudAppRPCs/011_UpdateAppList_NewCloudApplication.lua
```

**Pull request merged**